README.md

```diff
@@ -43,7 +76,7 @@ ck0i0krq457c7bgte4l0:
 Read the `vault-initialization` secret content:
 
 ```bash
-user@ubuntu:~$ juju show-secret ck0i0krq457c7bgte4l0 --reveal
+user@ubuntu:~$ juju show-secret ckp5ufc1tqvs77qu9gag --reveal
```

Comment:
I'm not sure that it is necessary to change the example. In any case, if you do it make sure to leverage the same example values here and under the `juju secrets --format=yaml` command.

Reply:
Done